terra-batch-libchart/values.yaml

```diff
@@ -32,7 +32,7 @@ exports:
 
     wds:
       name: wds
-      image: gcr.io/broad-dsde-dev/workspacedata
+      image: us.gcr.io/broad-dsp-gcr-public/terra-workspace-data-service:6786552
```

Comment:
It would be nice to make this semantic version if one exists?

Reply:
it does not :( 

https://broadinstitute.slack.com/archives/C03T815L316/p1663958552690649